src/main/java/org/folio/rest/utils/AutomaticFeeFineInitializer.java

```diff
@@ -0,0 +1,62 @@
+package org.folio.rest.utils;
+
+import java.util.Date;
+import java.util.List;
+import java.util.UUID;
+
+import org.folio.rest.jaxrs.model.Feefine;
+import org.folio.rest.jaxrs.model.Metadata;
+import org.folio.rest.persist.PostgresClient;
+
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Handler;
+
+import static java.util.Arrays.asList;
+
+public class AutomaticFeeFineInitializer {
+
+  private static final String FEEFINES_TABLE_NAME = "feefines";
+  private PostgresClient postgresClient;
+  private Handler<AsyncResult<String>> replyHandler;
+  private List<String> feefines;
```

Comment:
Could any of these be `final`? It seems like `postgresClient` and `feefines` are never modified. Can `replyHandler` be passed to a constructor as well rather than set with a setter method?